src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -137,6 +137,39 @@
 		}
 	},
 
+	"PORTCHANNEL_INTERFACE_IP_ADDR_TEST": {
+		"sonic-portchannel:sonic-portchannel": {
+			"sonic-portchannel:PORTCHANNEL": {
+				"PORTCHANNEL_LIST": [{
+					"name": "PortChannel0001",
+					"admin_status": "up"
+				}]
+			},
+			"sonic-portchannel:PORTCHANNEL_INTERFACE": {
+				"PORTCHANNEL_INTERFACE_IPPREFIX_LIST": [{
+					"name": "PortChannel0001",
+					"ip_prefix": "1.1.1.1/24"
+				}]
+			}
+		}
+	},
+
+	"PORTCHANNEL_INTERFACE_IP_ADDR_ON_NON_EXIST_PO": {
+		"sonic-portchannel:sonic-portchannel": {
+			"sonic-portchannel:PORTCHANNEL": {
+				"PORTCHANNEL_LIST": [{
+					"name": "PortChannel0001"
```

Comment:
Better to add admin_status, since it is mandatory. Libyang may throw error about mandatory before leafref and the test case may fail. So better to keep one failure scenario in the test. Thx.